packages/@okta/vuepress-site/guides/sign-into-spa/section10/react/getuserinfo.md

```diff
@@ -1,4 +1,4 @@
-User info ("claims") is returned from the `getUser()` method on the `Auth` object. The `Auth` object is made available in your components as `props.auth` via the `withAuth` Higher Order Component (HOC).
+Your code can get the user's profile using the `getUser()` method on the `Auth` object. The `Auth` object is made available in your components as `props.auth` via the `withAuth` higher-order component.
 
 ```javascript
```

Comment:
@aarongranick-okta Ditto to my comment on the Angular content - is there a small example of rendering the user's name that we could put here?

You had an example in section 7 but I removed it since we hadn't talked about this step yet. That could fit here?